css-layout-classes/styles.css

```diff
@@ -0,0 +1,51 @@
+* {
+  font-family: sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-third {
+  width: 100%;
+  flex-wrap: nowrap;
+  flex-basis: calc(100% / 3);
+  text-align: center;
+}
+
+.column-fourth {
+  flex-basis: 25%;
+}
+
+.characters {
+  max-height: 100%;
+  flex-basis: auto;
```

Comment:
This `flex-basis` isn't doing anything because it's parent isn't a flex parent, so this should be deleted